.gitignore

```diff
@@ -26,3 +26,4 @@ database/data
 *.pyc
 /devtools
 .python-version
+.vscode
```

Comment:
Do you mind adding a newline at the end?

Reply:
sure. Just added